opentreemap/manage_treemap/routes.py

```diff
@@ -1,19 +1,21 @@
 # -*- coding: utf-8 -*-
+from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
-from __future__ import division
 
 from functools import partial
 
 from django_tinsel.decorators import route, render_template, json_api_call
 from django_tinsel.utils import decorate as do
 
+import manage_treemap.views.management as views
+import manage_treemap.views.photo as photo_views
+
+from importer.views import list_imports
+from otm_comments.views import comment_moderation
```

Comment:
We are importing `comment_moderation` here, but then assigning `comment_moderation =` below. It would be more clear and consistent with the way we import the photo review view by switching to

```python
import otm_comments.views as comment_views
```

```python
comment_moderation = admin_route(
    GET=do(render_template('manage_treemap/comment_moderation.html'),
           comment_views.comment_moderation)
```